pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -207,6 +212,46 @@ func eventFromChannel(c chan string, testName string, wantEvents []string) error
 	return nil
 }
 
+// getTaskRunCreations will look through a set of actions to find all task run creation actions and return the set of
+// them. It will fatal the test if none are found.
+func getTaskRunCreations(t *testing.T, actions []ktesting.Action) []*v1beta1.TaskRun {
+	outputs := []*v1beta1.TaskRun{}
+	for _, a := range actions {
+		if action, ok := a.(ktesting.CreateAction); ok {
+			if output, ok := action.GetObject().(*v1beta1.TaskRun); ok {
+				outputs = append(outputs, output)
+			}
+		}
+	}
+
+	if len(outputs) > 0 {
+		return outputs
+	}
+	t.Log("actions", actions)
+	t.Fatal("failed to find an action that created a taskrun")
+	return nil
+}
+
+// getPipelineRunUpdates will look through a set of actions to find all PipelineRun creation actions and return the
+// set of them. It will fatal the test if none are found.
+func getPipelineRunUpdates(t *testing.T, actions []ktesting.Action) []*v1beta1.PipelineRun {
+	outputs := []*v1beta1.PipelineRun{}
```

Comment:
Same here : We should put a `t.Helper()` here so that the error report points to the caller instead of the `t.Fatal` in this function